swift_browser_ui/ui/login.py

```diff
@@ -80,11 +86,20 @@ async def oidc_end(request: aiohttp.web.Request) -> aiohttp.web.Response:
         "state": oidc_result["state"],
         "access_token": oidc_result["token"],
     }
+    request.app["Log"].info(session["oidc"])
```

Comment:
This should be removed, but it's the way to see the content from the OIDC provider.

Reply:
Ok, though it might be fit to lower the logging level to `debug`